Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md

```diff
@@ -175,33 +179,71 @@ To deploy a virtual machine with compute services you can edit one of windows or
-| Instances | Specify the number of instances to spawn. If you increase it to a value higher than 1, a **High Availability** toggle will appear below that you can enable to use Azure availability sets for high availability support. Note that at this time, we don’t autoscale or load balance instances. To enable that, you have to manually configure these options in [Azure](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-how-to-scale-portal) |
+| Instances | Specify the number of instances to spawn. If you increase it to a value higher than 1, a **High Availability** toggle will appear below that you can enable to use Azure availability sets for high availability support. |
+| ScaleSet  | Specify if the instance/s should be created into a ScaleSet resource. When switched on, it will enable auto-scaling and load-balancer sections (see below)  |
 | Delegate Management  | Delegate management to CenturyLink.  |
 
 When you increase the number of instances, the High Availability toggle appears:
 
-![High Availability toggle visible](../../images/cloud-application-manager/microsoft-azure-compute-deployment-options-4-ha.png)
+![High Availability toggle visible](../../images/cloud-application-manager/microsoft-azure/compute-deployment-options-ha.png)
 
 For more information, see the [Azure help](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/tutorial-availability-sets#availability-set-overview)
 
 #### Network
 
+![Microsoft Azure Deployment Options - Network](../../images/cloud-application-manager/microsoft-azure/compute-deployment-options-network.png)
+
 | Option | Description |
 |--------|-------------|
 | Virtual Network | Choose the network to deploy to among the list of available networks. |
 | Subnet | This subnet is the resource related to  the virtual machine's network interface. Actually a virtual network is not used at deployment time. |
-|Security Group | Filter incoming and outgoing traffic for the virtual machine based on a set of rules. Multiple security groups in a zone can be selected for a virtual machine. For more information, see [Security Groups](https://docs.microsoft.com/en-us/azure/virtual-network/security-overview). |
-|Public IP Address   | The public IP Address exposes our server to the public internet where other applications can access it.|
+|Security Group | Filter incoming and outgoing traffic for the virtual machine based on a set of rules. Multiple security groups in a zone can be selected for a virtual machine.  For more information, see [Security Groups](https://docs.microsoft.com/en-us/azure/virtual-network/security-overview).  This option will not appear if you choose to use a **ScaleSet** feature above.|
+|Public IP Address | The public IP Address exposes our server to the public internet where other applications can access it. If loadbalancing is configured, this public IP will be associated with it, instead of associating it directly with the instances. |
```

Comment:
Here we can mention that in case of loadbalancers a new public ip will be created and associated to them